e2e/tests/text/text-editing/index.spec.ts

```diff
@@ -24,7 +19,7 @@ test('textbox typing and resizing', async ({ page }) => {
 
   await page
     .locator('textarea')
-    .type(
+    .pressSequentially(
```

Comment:
type method is deprecated and pressSequentially is the new recommended one by the docs